api/accounts/views.py

```diff
@@ -64,11 +64,10 @@ class ProfileList(generics.ListAPIView):
 
 
 class ProfileDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Profile.objects.all()
```

Comment:
What was the thinking behind changing this? The detail page should show the signed in users profile (which is what the deleted code did). Does this work the same way?

Reply:
For some reason with the old code I could not see the detail page for every user but this change I don't believe is necessary, I only used it to see the details of other users. I can change it back.